### PR TITLE
change the A record to an CNAME record

### PR DESCRIPTION
**Pull request by ploszukiwacz**

### CODE REVIEW

File: domains/ploszukiwacz.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "record": {
     "A": [
-      "45.131.64.15"
+     172.66.47.187
```
Comment (CuteDog5695) [inline domains/ploszukiwacz.json:11]:
```suggestion
     "172.66.47.187"
```

**Review by iostpa — CHANGES_REQUESTED** [review]
Hallo, ya file is incorrect.

File: domains/ploszukiwacz.json (hunk 1 above)
Comment (PlOszukiwaczDEV) [reply domains/ploszukiwacz.json]: i did this

**Review by iostpa — APPROVED** [review]
A

**Comment by PlOszukiwaczDEV** [comment]
and now i wait for is-a.dev to build

**Pull request merged**